docs/troubleshooting.adoc

```diff
@@ -341,16 +341,16 @@ Taken from https://support.quest.com/SolutionDetail.aspx?id=SOL86281
 
 == Windows installation related issues
 
-=== Configuration of the agent if the Jenkins master  changed the address
+=== Configuration of the agent if the Jenkins controller  changed the address
 
 Make sure to check out logs of the starting service. 
-If you are changing configuration of the jenkins it is possible that the agent tries to connect to old masters address.
+If you are changing configuration of the jenkins it is possible that the agent tries to connect to old controllers address.
 To fix this:
 
 . on the agent: stop jenkins windows service (if not already dead)
-. on the master: go to Jenkins > Manage Jenkins > Configure System, and
+. on the controller: go to Jenkins > Manage Jenkins > Configure System, and
 copy value from 'Jenkins URL' parameter
-. on the agent: edit jenkins-slave.xml and alter there service/arguments
+. on the agent: edit jenkins-agent.xml and alter there service/arguments
```

Comment:
are you sure that this property file exists ? `jenkins-agent.xml`

Reply:
I checked the source code and some other resources bundle, and it seems this file is already mentioned and searched

Reply:
ok then.